homeassistant/components/fastdotcom/strings.json

```diff
@@ -1,8 +1,13 @@
 {
-  "services": {
-    "speedtest": {
-      "name": "Speed test",
-      "description": "Immediately executs a speed test with Fast.com."
+  "config": {
+    "step": {
+      "user": {
+        "description": "Do you want to start the setup? The initial setup will take about 30-40 seconds."
```

Comment:
Seems strange? Setting up the config entry is pretty instant. Or you mean first data refresh which is not relevant here.

Reply:
Yes, I thought it might be helpful to notify the user that the initial data refresh takes longer, since it will actually perform it twice. Each call takes around 15-16 seconds. So to hint out that HA isn't crashing, just a headsup the initialization process takes a bit more time.